### PR TITLE
RET-4516

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3917

### Change description ###

Revert renovate json update

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by JackReevies — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -247,7 +247,7 @@ dependencies {
 
   implementation group: 'org.elasticsearch', name: 'elasticsearch', version: '8.9.1'
   implementation group: 'org.springframework.boot', name: 'spring-boot-devtools', version: '2.7.15'
-  implementation 'org.json:json:20230618'
+  implementation 'org.json:json:20230227'
```
Comment (marcjudge) [inline build.gradle:250]: Do you want to add something to dependabot config to fix to this version?

Comment (marcjudge) [reply build.gradle:250]: renovate I mean

Comment (hsjhita) [reply build.gradle:250]: hopefully shouldn't need this PR as PR-506 should fix it but if we do, can look at renovate changes to prevent this upgrade

**Comment by hsjhita** [comment]
Closed as https://github.com/hmcts/et-sya-api/pull/506 worked

**Pull request closed without merging**